app/models/user.rb

```diff
@@ -118,6 +118,10 @@ class User < ApplicationRecord
       user.update(voting_policy_agreement: false)
     end
 
+    after_transition on: all - [:key_member] do |user, _|
```

Comment:
I'm not familiar with this syntax but it's cool :)

Reply:
I read the docs. I'm very grown up. 😹